examples/oh_experiments/parallel_adder_sweep.py

```diff
@@ -1,60 +0,0 @@
-# Copyright 2020 Silicon Compiler Authors. All Rights Reserved.
-
```

Comment:
Motivation for deleting example?

Reply:
The example has been broken for a long time (initializes Chip object without design) and isn't referenced in docs/CI. 

I'm also not sure it adds much since we already have a multiprocessing example (examples/heartbeat/parallel.py) and a sweep example (examples/oh_experiments/adder_sweep.py) that are both up-to-date and tested.

Reply:
Although we could rewrite this very nicely using a multi-index approach once we have ['option', 'param'] be overridable on a step/index basis.

Reply:
Yes, you are right, the heartbeat/parallel.py illustrates the pattern I was looking for.